src/openzaak/components/zaken/openapi.yaml

```diff
@@ -24,13 +24,9 @@ info:
     \n**Main resource**\n\n`zaak`\n\n\n\n**Kenmerken**\n\n* `bronorganisatie`: Het\
     \ RSIN van de Niet-natuurlijk persoon zijnde de organisatie die de zaak heeft\
     \ gecreeerd. Dit moet een geldig RSIN zijn van 9 nummers en voldoen aan https://nl.wikipedia.org/wiki/Burgerservicenummer#11-proef\n\
-    * `zaaktype`: URL-referentie naar het ZAAKTYPE (in de Catalogi API) in de CATALOGUS\
-    \ waar deze voorkomt\n* `vertrouwelijkheidaanduiding`: Aanduiding van de mate\
-    \ waarin het zaakdossier van de ZAAK voor de openbaarheid bestemd is.\n\n**Resources\
-    \ en acties**\n- `zaak`: create, update, destroy\n- `status`: create\n- `zaakobject`:\
-    \ create\n- `zaakinformatieobject`: create\n- `zaakeigenschap`: create\n- `klantcontact`:\
-    \ create\n- `rol`: create\n- `resultaat`: create, update, destroy\n- `zaakbesluit`:\
-    \ create\n\n\n**Handige links**\n\n* [Documentatie](https://zaakgerichtwerken.vng.cloud/standaard)\n\
+    * `zaaktype`: URL-referentie naar het ZAAKTYPE (in de Catalogi API).\n* `vertrouwelijkheidaanduiding`:\
```

Comment:
stuk notificaties docs is weg

Reply:
Ik zag het maar begreep niet hoe dat door mij kwam dus nam aan dat iemand anders een change had gedaan.

Ik ga er achter aan.

Reply:
This seems to be an issue in the master branch as well. It was just not updated with the schema's.

I'll create an issue for this.